.circleci/config.yml

```diff
@@ -351,11 +353,12 @@ jobs:
       - attach_workspace:
           at: ~/repos
       - *restore-go-mod-cache
+      - run: make prepare-ethersjs-project
         # Run the tests with coverage parse the coverage and output the summary
       - run:
           name: Run tests and print coverage summary
           command: |
-            go test -coverprofile cov.out -coverpkg ./consensus/istanbul/... ./e2e_test 
+            go test -v -coverprofile cov.out -coverpkg ./consensus/istanbul/... ./e2e_test 
```

Comment:
Do you want to keep the added `-v`?

Reply:
Yes that was intentional, we want to have the output in CI so we can more easily analyse failures.